### PR TITLE
[ESIMD] Update README.md

### PR DESCRIPTION
**Pull request by kbobrovs**
Signed-off-by: Konstantin S Bobrovsky <konstantin.s.bobrovsky@intel.com>

### CODE REVIEW

**Comment by bader** [comment]
@kbobrovs, I canceled Lit_With_Cuda job to reduce the queue. Missing results for this job should not block the merge.

File: sycl/doc/extensions/experimental/SYCL_EXT_INTEL_ESIMD/README.md [hunk 1]
```diff
@@ -19,76 +19,80 @@ trivial example which adds elements of two arrays and writes the results to the
...
+objects and `copy_to` intrinsics are used which are avaiable only in the ESIMD extension.
 Full runnable code sample can be found on the
 [github repo](https://github.com/intel/llvm-test-suite/blob/intel/SYCL/ESIMD/vadd_usm.cpp).
 
 #### Compiling and running ESIMD code.
 
-A code, which uses the ESIMD extension can be compiled and run using the same
-options as the regular SYCL code, e.g.:
+Compiling and running code that uses the ESIMD extension is the same as compiler and
+running code that uses the standard SYCL:
```
Comment (gmlueck) [inline sycl/doc/extensions/experimental/SYCL_EXT_INTEL_ESIMD/README.md:53]:
```suggestion
Code that uses the ESIMD extension can be compiled and run using the same commands
as standard SYCL:
```

**Review by gmlueck — APPROVED** [review]

**Review by kychendev — APPROVED** [review]

File: sycl/doc/extensions/experimental/SYCL_EXT_INTEL_ESIMD/README.md [hunk 2]
```diff
@@ -19,76 +19,80 @@ trivial example which adds elements of two arrays and writes the results to the
...
 
 #### Compiling and running ESIMD code.
 
-A code, which uses the ESIMD extension can be compiled and run using the same
-options as the regular SYCL code, e.g.:
+Code that uses the ESIMD extension can be compiled and run using the same commands
+as standard SYCL:
 
+To compile using the open-source Intel DPC++ compiler:
 > `$ clang++ -fsycl vadd_usm.cpp`
```
[the remark this answers is not in the document]
Comment (kbobrovs) [reply sycl/doc/extensions/experimental/SYCL_EXT_INTEL_ESIMD/README.md:56]: In open-source we do have clang++

**Comment by kbobrovs** [comment]
> @kbobrovs, I canceled Lit_With_Cuda job to reduce the queue. Missing results for this job should not block the merge.

@bader, thanks, but there was a change to address review comments. Please do it again if CUDA testing happens to stall again.

**Pull request merged**